packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_model.dart

```diff
@@ -7,7 +7,7 @@ import 'package:devtools_app_shared/utils.dart';
 import 'package:flutter/material.dart';
 
 import '../../shared/primitives/utils.dart';
-import '../../shared/table/table.dart';
+import '../../shared/table/table.dart' as table;
```

Comment:
why the prefix? Is there a class name conflict?

Reply:
I think this is unrelated, will remove